packages/components/modules/comments/Comments/index.tsx

```diff
@@ -10,7 +10,6 @@ import { CommentsProps } from './types'
 
 const Comments: FC<CommentsProps> = ({
   target: targetRef,
-  profileId,
```

Comment:
we should also remove the `profileId` from `CommentsProps` and from its storybook